README.rst

```diff
@@ -47,6 +49,14 @@ Getting Started
 
 .. _Running server on device: #browsing-server-on-device
 
+Issues
+---------------
+For questions on development for the App you can post a topic here 
+https://groups.google.com/forum/#!forum/zulip-android
+
+For questions on development for the Server you can post a topic here 
+https://groups.google.com/forum/#!forum/zulip-devel
```

Comment:
> (2) Should we also point people at the Android stream on zulip.tabbott.bet?

Its totally upto you? 
New contributors do face problem in setting up a development server, it could be easier for them as well!